string greeting1
int32 number
uint32 unumber
string greeting2
float32 fnumber
bool TrueOrFalse